tests/labs/lie/functional/common.py

```diff
@@ -126,3 +126,60 @@ def func(matrix: torch.Tensor):
         return module._left_project_autograd_fn(group, matrix[sels, ..., sels, :, :])
 
     return func
+
+
+def check_jacrev_unary(group_fns, dim, batch_size, name):
```

Comment:
I think we add another two arguments to specify the types (Euclidean/Riemannian) of the input and output. In that case, we could set the tests automatically.

Reply:
In the old test code I added [this](https://github.com/facebookresearch/theseus/blob/main/tests/labs/lie/functional/common.py#L13), so the same test code can be used for all operators. I didn't add something like this here because (for now) we are only testing 4 operators.

Reply:
That makes sense. All the tests look good to me.

> In the old test code I added [this](https://github.com/facebookresearch/theseus/blob/main/tests/labs/lie/functional/common.py#L13), so the same test code can be used for all operators. I didn't add something like this here because (for now) we are only testing 4 operators.